tests/infra/ccf.py

```diff
@@ -185,8 +185,11 @@ def start_and_join(self, args):
         self.wait_for_all_nodes_to_catch_up(primary)
         LOG.success("All nodes joined network")
 
+        self.add_users(primary, initial_users)
```

Comment:
This and `open_network()` a few lines below makes the CCF network startup procedure a little long, about 2-3 extra seconds per end-to-end test.

Reply:
That seems like a lot, do we know why?

Reply:
Probably because each user needs to be added via proposal and voted for by the members

Reply:
This but also the fact that we wait for proposals to be globally committed (for good reasons). I've added an option to issue `mkSign` when waiting for a global commit. This shaves off 1.5 seconds of the network opening phase, which I think is as good as it gets.